### ISSUE TITLE
DeepAR-Electricity.ipynb issues

### ISSUE THREAD

**Issue by aaronmarkham** [aws/amazon-sagemaker-examples#1352]
https://github.com/awslabs/amazon-sagemaker-examples/blob/master/introduction_to_amazon_algorithms/deepar_electricity/DeepAR-Electricity.ipynb

* error
```
SyntaxError: from __future__ imports must occur at the beginning of the file
```
(This is easily fixed by moving this line to the top of the block.)

* error 2
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-18-bf11b819691d> in <module>
      4         "target": ts[start_dataset:end_training - 1].tolist()  # We use -1, because pandas indexing includes the upper bound
      5     }
----> 6     for ts in timeseries
      7 ]
      8 print(len(training_data))

<ipython-input-18-bf11b819691d> in <listcomp>(.0)
      4         "target": ts[start_dataset:end_training - 1].tolist()  # We use -1, because pandas indexing includes the upper bound
      5     }
----> 6     for ts in timeseries
      7 ]
      8 print(len(training_data))

pandas/_libs/tslibs/c_timestamp.pyx in pandas._libs.tslibs.c_timestamp._Timestamp.__sub__()

pandas/_libs/tslibs/c_timestamp.pyx in pandas._libs.tslibs.c_timestamp._Timestamp.__add__()

TypeError: Addition/subtraction of integers and integer-arrays with Timestamp is no longer supported.  Instead of adding/subtracting `n`, use `n * obj.freq`
```
Changing the `-1` to `end_training.freq` seems to do the trick, but then there are more similar errors that aren't as easily resolved.

* error 3
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-31-1620325d0e66> in <module>
      6         "target": ts[start_dataset:end_training + k * prediction_length].tolist()
      7     }
----> 8     for k in range(1, num_test_windows + num_test_windows)
      9     for ts in timeseries
     10 ]

<ipython-input-31-1620325d0e66> in <listcomp>(.0)
      7     }
      8     for k in range(1, num_test_windows + num_test_windows)
----> 9     for ts in timeseries
     10 ]
     11 print(len(test_data))

pandas/_libs/tslibs/c_timestamp.pyx in pandas._libs.tslibs.c_timestamp._Timestamp.__add__()

TypeError: Addition/subtraction of integers and integer-arrays with Timestamp is no longer supported.  Instead of adding/subtracting `n`, use `n * obj.freq`
```

**Comment by vafl** [comment]
I think this is fixed in #1332

**Comment by seanpmorgan** [comment]
Closing. As mentioned this should be fixed now
